client/spec/helpers/workspace.js

```diff
@@ -67,6 +68,94 @@ function Workspace() {
     };
 
     /**
+     * Show name list from right menu
+     *
+     * @param {string} name
+     */
+    this.showList = function(name) {
+        this.sideMenu.element(by.css('[title="' + name + '"]')).click();
+    };
+
+    /**
+     * Show the name highlight list
+     *
+     * @param {string} name
+     */
```

Comment:
Just a remark here: docstrings can also be used for automatically generating documentation, and since the source code is not included there, there is a merit in writing them even if one thinks that the code itself is too straightforward for that.

(it's another story, though, whether we actually make use of that...)